src/scripts.js

```diff
@@ -65,10 +74,22 @@ function displayAllRecipes() {
           <h2>${oneRecipe.name}</h2>
         </section>
     `
-    allRecipes.innerHTML += miniRecipe;
-  }
+    allRecipesView.innerHTML += miniRecipe;
+  } 
+}
+
+function displayPantryView() {
```

Comment:
love these functions! We might see if we can refactor it since we are using the same methods to change class in each function.